### DISCUSSION TITLE
Información Sobre las Traducciones

### DISCUSSION THREAD

**Discussion by mora145** [Ludeon/RimWorld-SpanishLatin#73] [General]
Lo traje de la traducción castellana. Me parece es que es una buena "pizarra" de las cosas que faltan ademas de dar algunas instrucciones en cuanto a las traducciones y el orden técnico que deberíamos seguir para no confundirnos entre versiones a la hora de traducir. He adaptado el texto a nuestras necesidades.
Escrito por: Ghrull
-
Saludos, hago una pequeña guía/sumario de la traducción de Rimword al castellano para ayudar a los nuevos traductores y que no estén tan perdidos como lo estuve yo en su día.

Herramientas para traducir y Discord para comunícanos mejor:
Discord: Servidor de Discord para que podamos comunicarnos de manera mas cómoda y eficiente.
Enlace: https://discord.gg/FDCMT2S

Rimtrans: Esta herramienta genera los archivos *.xml en ingles, a partir de esos archivos hacemos la traducción.
Enlace: https://github.com/RimWorld-zh/RimTrans

Notepad ++: Con esta herramienta se edita y traduce los archivos *.xml. Hay otros editores y formas de editar, pero personalmente creo que este editor es el mejor y es gratis.
Enlace: https://notepad-plus-plus.org/

Deepl: El mejor traductor online que hay actualmente, es perfecto para consultar palabras o frases de difícil traducción.
Enlace: https://www.deepl.com/translator

Correctores de ortografía y gramática: Estas herramientas online pueden ayudar a corregir los textos ya traducidos antes de ser subidos a Github.
Enlace: https://languagetool.org/es/
Enlace: https://spanishchecker.com/

Estado de la traducción (13/07/2018):
Carpeta Backstories: Solo contiene un archivo (Backstories.xml) que hay que revisarlo entero ya que muchas traducciones están mal, esta desactualizado y hay que añadir la dualidad de genero. Creo que lo mejor seria traducir Backstories.xml directamente del archivo original en ingles. Sobre la dualidad de genero, este seria un ejemplo:

Actualmente: [PAWN_nameDef] (Por ejemplo, Laura) trabajaba como granjero y era muy feo.
Sustituir por: [PAWN_nameDef] (Por ejemplo, Laura) trabajaba como granjero(a) y era muy feo(a).

Siempre usaremos los paréntesis en lugar de barras puesto que ya se hizo así en los otros textos del juego.

Carpeta DefInjected: Esta carpeta es la que mas textos contiene del juego, esta traducida prácticamente entera pero hay que revisar muchas cosas:

-Las carpetas que a su derecha tienen el texto  "Traducción 1.0.1938 (development build)" tienen que ser revisadas ya que podrían haber añadido alguna nueva linea o cambio del código, por ejemplo, en ThingDef/Plants_Cultivated_Farm.xml hay que añadir el cacao.

-Las carpetas de reglas, por ejemplo RulePackDef, a no ser que sepan lo que hacen, recomiendo no modificarlas. Aun faltan algunas traducciones y adaptar el texto a los femeninos y masculinos (Ejemplo: atacandolo(a), golpeandolo(a), etc). Actualmente (En la version 1.0.1967) hay un problema con las etiquetas ORIGINALTARGET_definite y aquellas etiquetas que especifican daño la parte del cuerpo. Lo dejaremos así hasta la siguiente versión, a ver si el problema se soluciona.  

-El resto de carpetas, sobre todo las que no se actualizan desde hace mucho tiempo tienen que ser revisadas y actualizadas (código y texto si fuese necesario).

Carpeta Keyed: En su version actual (1.0.1967) terminada.

Carpeta Strings: Esta carpeta no esta traducida, yo la actualice a la versión 1.0 en ingles porque la versión que estaba traducida era muy antigua y daba muchos problemas. Son traducciones pequeñas y algunos archivos no necesitan ser traducidos.

LEER PD AL FINAL

Consejos:
-Cuando se vaya a actualizar un archivo que este muy anticuado lo mejor es traducir directamente el archivo actualizado de la versión en ingles y luego subir ese archivo (borrar el antiguo y subir el nuevo).

-Cuando se suba o modifique algún archivo, en el apartado "Commit changes" seria de gran ayuda escribir la versión del archivo y si es posible una descripción de lo que se a modificado, por ejemplo "Traducción 1.0.1938", de esta manera estará todo mas organizado y sera mas fácil saber que carpetas y archivos necesitan ser actualizados o revisados.

-REVISAR BIEN LAS FALTAS DE ORTOGRAFÍA Y NO HACER UN CORTA Y PEGA DEL TRADUCTOR.
La herramienta que uso para generar textos (Rimtrans) es de la alpha 18 y da errores con la versión 1.0 (espero que pronto la actualicen). De todas maneras pienso seguir traduciendo Rimworld en mis ratos libres pero seria de gran ayuda que mas traductores se uniesen. Saludos¡¡

@Destiny609 @Andres6936 @Etaglibash

**Comment by NNeoNeTT** [comment]
@GoldenMaur1 Buenas colega, soy Destiny como lo dice mi nick xD, no lo confundas, soy uno de los traductores junto a Mora que seguimos activos, y te doy la bienvenida, antes que todo te pediría que consultes antes de hacer algo, ya que nos ha tomado bastante tiempo a mi y a los demás traductores dejar la traducción hasta este punto, y como veras no hay  actividad desde que salio la B19 ya que solo quedan traducir unas cuantas Rules y como no son de tanta importancia las dejamos, cualquier consulta me la puedes hacer al siguiente correo wizardofdestiny9@gmail.com

Desde ya.
Saludos

**Comment by zerstrick** [comment]
No había visto esto este tema @mora145  mucha de las cosas que se comentan al parecer están desactualizadas, por ejemplo ahora en lo posible estoy haciendo reglas que se colocan o añaden en el archivo RulePacks_Global.xml esto con el objeto de añadir nuevos archivos para la distinción de género, en la capeta Strings, por ejemplo: si vamos a Strings/Words/Nouns se han creado nuevos archivos para dar distinción de genero, especialmente se conserva la forma original del archivo para masculino y se crea un nuevo archivo para femenino, [ejemplo: **Color.txt**(forma original) **Color_Fem.txt**(forma variante)] así se puede hablar de rojo y roja, negro y negra, que serán usados en distintos lugares del juego o para complementar otros nombres que se han añadido usando la misma forma, como el caso de las descripciones de arte: "sentado frente a una mesa roja" también estoy usando lo que en el foro se planteo para la distinción de sexo en algunos caso donde aplique y lo permita el juego con el nuevo sistema,  {0_gender ? o : a} esto para el caso de las historias (archivo que terminaré pronto) donde estoy evitando el uso del (a) "era un(a) niño(a) hermoso(a)" y se sustituye por "era un{PAWN_gender ?  : a} niñ{PAWN_gender ? o : a} hermos{PAWN_gender ? o : a}" que dará como resultado dependiendo del sexo del peón (PAWN) dos textos bien definidos "era un niño hermoso" "era una niña hermosa" más información de este cambio lo pueden consultar aquí: https://ludeon.com/forums/index.php?topic=43979.0 en ese mismo archivo se historias también se está añadiendo títulos femeninos donde aplique, todo con el objeto de que si sale un peón femenino tanto la descripción como el título estén acorde. Me ha tomado mucho tiempo ir actualizando el gran trabajo que hicieron, también corregir muchas cosas que estaban por pulir, mejorar la semántica para los distintos nombres que que había para un mismo objeto, pero su gran trabajo me sirvió de guía inicial para continuar con lo que ya se había hecho, espero poder terminar todo lo que falta dentro de muy poco :D

**Comment by Andres6936** [comment]
@zerstrick Muchísimas gracias desde ahora, es incalculable el trabajo que estas haciendo con las traducciones. Te agradezco enormemente el trabajo y tiempo que has investido en este proyecto. Muchas gracias @zerstrick y @mora145

**Comment by mora145** [comment]
@Andres6936 Gracias tambien por tus aportes man. Entre todos, y los que estuvieron antes que nosotros tambien, sacamos adelante esta traducción.
@zerstrick Entiendo, que bueno que se reparó eso de los géneros. Creo que eran una de las cosas mas importantes que faltaban corregirse. 
Sobre la etiqueta reductora, creo que esta: &lt;size=24&gt;**Notes**&lt;/size&gt;, donde "Notes" es el texto a reducirse, y "size=24" es el tamaño de la letra.

**Comment by mishchenkoy** [comment]
👍👌
